pkg/kubelet/apis/podresources/types.go

```diff
@@ -34,5 +37,6 @@ type PodsProvider interface {
 
 // CPUsProvider knows how to provide the cpus used by the given container
 type CPUsProvider interface {
-	GetCPUs(podUID, containerName string) []int64
+	GetCPUs(podUID, containerName string) cpuset.CPUSet
+	GetAllocatableCPUs() cpuset.CPUSet
```

Comment:
Likewise here -- this is where I would have expected it to be translated from a `CPUSet` to whatever type the `podresourcesapi` requires.

Reply:
ack, same as above.